hq/README.md

```diff
@@ -29,32 +29,31 @@ N.B. If you are using macOS you may need to install `coreutils` before you can r
 ### Getting started
 
 ### AWS Configuration
-Go to AWS IAM console (security account) and clone `security-hq-dev` user to security-hq-<yourname>
-Create an Access key from the Security Credentials tab and save the access key under `security-test` section in your `~/.aws/credentials` file
-You should have an entry looks like below
+
+Using the Security Admin account, go to AWS IAM console and clone the `security-hq-dev` user to `security-hq-<yourname>`.
+
+At the last step of the creation of this new user, you will be presented with an *Access key ID* and a *Secret access key* that you are going to use during the next step. (Alternatively, you can create those credentials from an existing user from the Security Credentials tab.)
+
+Create the file `~/.aws/credentials` with the following contents
+
 ```
 [security-test]
 aws_access_key_id = xxxxxxxx
 aws_secret_access_key = yyyyyyyy
 ```
+
+... where you put the *Access key ID* and *Secret access key* of the previous step. If you already had this file, you should just add this section.
+
 Create the entry below in `~/.aws/config` file.
+
 ```
 [profile security-test]
 region = eu-west-1   
 ```
 
-Ensure that a file named `security-hq.local.conf` exists in your .gu directory, and that it contains the required information, otherwise the project may not run.
-Ensure you have `security` account listed in your `security-hq.local.conf` in `AWS_ACCOUNTS` config, like below.
+The next file is `~/.gu/security-hq.local.conf`, which should be given by one of your team members. Once you have it, make sure to update the value of key `GOOGLE_SERVICE_ACCOUNT_CERT_PATH`. (The rest of the file is common across team members.)
```

Comment:
` (The rest of the file is common across team members.)` Not strictly true... I've tailored mine!

I would remove:  `(The rest of the file is common across team members.)`

Reply:
Corrected!